packages/js/data/src/products/action-types.ts

```diff
@@ -1,4 +1,5 @@
 export enum TYPES {
+	CREATE_PRODUCT_START = 'CREATE_PRODUCT_START',
```

Comment:
I added these events for any of the actions, as there was no way to tell when a product was being updated, created, or deleted. The `isResolving` or `hasFinishedResolution` only work for resolvers/selectors I guess.

In the reducer these events get toggled within the `pending` object (see the `reducer.ts` changes for this) and set back to `false` when the success or error actions are triggered.